assets/scripts/expressions/em_javascript.js

```diff
@@ -1021,7 +1021,10 @@ function LEMval(alias)
                     return "";
                 }
 
-                var checkNumericRegex = new RegExp(/^(-)?[0-9]*(,|\.)[0-9]*$/);
+                var isNumericRegex = new RegExp(/^(-)?[0-9]*(\.)[0-9]*$/);
```

Comment:
This is used more than once. Maybe we can put it on a function named like `getNumericValidatioRegex`?

Reply:
Not sure if this is a must for merging... Will leave it up to you @Shnoulle

Reply:
> This is used more than once. Maybe we can put it on a function named like `getNumericValidatioRegex`?

I just fix here, not rewrite.

Reply:
> Not sure if this is a must for merging... Will leave it up to you @Shnoulle

?